src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 198px;
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
```

Comment:
Check if the font-size, line-height, and font-weight match the design requirements. These values should typically be consistent across the project and may be defined in variables for reusability and maintainability.